PendingReleaseNotes

```diff
@@ -19,6 +19,11 @@
 * CephFS: The period specifier ``m`` now implies minutes and the period specifier
   ``M`` now implies months. This has been made consistent with the rest
   of the system.
+* RBD: The option ``--try-netlink`` to map rbd images to nbd devices has been
```

Comment:
Users know this option as `--options try-netlink` not `--try-netlink` -- the latter is accepted only by `rbd-nbd` tool, which we discourage users from interacting with directly.  I'd suggest making this a bit shorter:
```
* RBD: The ``try-netlink`` mapping option for rbd-nbd has become the default
  and is now deprecated. If the NBD netlink interface is not supported by the
  kernel, then the mapping is retried using the legacy ioctl interface.
```

Reply:
Also, why is this note added in the middle of the list instead of at the bottom?

Reply:
> Also, why is this note added in the middle of the list instead of at the bottom?

Help reduce merge conflicts with PRs that also amend PendingReleaseNotes.

Reply:
> > Also, why is this note added in the middle of the list instead of at the bottom?
> 
> Help reduce merge conflicts with PRs that also amend PendingReleaseNotes.

Have to rebase now due to merge conflicts in PendingReleaseNotes :)

Reply:
> Have to rebase now due to merge conflicts in PendingReleaseNotes :)

... and there is nothing wrong with that ;)